### PR TITLE
Various modifications

### PR DESCRIPTION
**Pull request by danielolsson100**
Added 
- ESM Status
- Frequency sensor as configurable values
- fixed some typo

### CODE REVIEW

File: custom_components/ferroamp/config_flow.py [hunk 1]
```diff
@@ -85,6 +86,10 @@ async def async_step_init(self, user_input=None):
         if precision_voltage is None:
             precision_voltage = 0
 
+        precision_frequency = self.config_entry.options.get(CONF_PRECISION_FREQUENCY)
+        if precision_frequency is None:
+            precision_frequency = 0
```
Comment (argoyle) [inline custom_components/ferroamp/config_flow.py:91]: Is zero decimals a good default value for frequency precision? Maybe we should go for 2 or more?

**Review by argoyle — CHANGES_REQUESTED** [review]
A small comment on the default precision and perhaps add a test for the new sensor? Just add a couple of asserts to the existing test for the ESM.

File: custom_components/ferroamp/config_flow.py (hunk 1 above)
Comment (danielolsson100) [reply custom_components/ferroamp/config_flow.py:91]: I agree. Go with 2 dec as a startingpoint for frequency defaitism

Comment (argoyle) [reply custom_components/ferroamp/config_flow.py:91]: Just make the change and push a new commit. The PR will be updated automatically.

**Comment by argoyle** [comment]
This closes #24

**Comment by argoyle** [comment]
Change the base branch to master as well

**Comment by argoyle** [comment]
Will split this into three separate PR's instead.

**Pull request closed without merging**